WordPress/JetpackStatsWidgets/Views/Cards/StatsValueView.swift

```diff
@@ -30,5 +30,6 @@ struct StatsValueView: View {
             .fontWeight(fontWeight)
             .foregroundColor(foregroundColor)
             .lineLimit(lineLimit)
+            .minimumScaleFactor(0.5)
```

Comment:
Why 0.5?

Reply:
0.5 is an arbitrary value, could be smaller or a bit larget as well.

Previously created rectangular widgets can now appear without a background on iPadOS Horizontal Lock Screen. Since they appear without a background, I increased font sizes to have less space and resemble the look of other widgets without a background.

However, when numbers become large, I want font to Value font to shrink, to be fully displayed.

Without `minimumScaleFactor`:
<img width="265" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/729028e3-63b5-4070-b0d0-69def7f192b6">

With `minimumScaleFactor`:
<img width="200" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/c1de6449-238b-4407-a585-d617d11a2ef5">